scripts/remove/nginx.sh

```diff
@@ -2,7 +2,24 @@
 
 systemctl stop -q nginx
 
-APT='nginx-extras nginx libnginx-mod-http-fancyindex ssl-cert php php-cli php-fpm php-dev php-xml php-curl php-xmlrpc php-json php-mcrypt php-opcache php-geoip php-xml php php-cli php-fpm php-dev php-xml php-curl php-xmlrpc php-json php-mcrypt php-opcache'
+codename=$(_os_codename)
+case $codename in
+    stretch)
```

Comment:
Stretch and bionic are EOL which means they will never even see any updates beyond the last EOL commit designated in swizzin. These cases are basically cruft